packages/@n8n/permissions/src/types.ts

```diff
@@ -1,19 +1,22 @@
 export type DefaultOperations = 'create' | 'read' | 'update' | 'delete' | 'list';
 export type Resource =
-	| 'workflow'
-	| 'tag'
-	| 'user'
+	| 'auditLogs'
+	| 'communityPackage'
 	| 'credential'
-	| 'variable'
-	| 'sourceControl'
+	| 'externalSecretsStore'
```

Comment:
This was renamed to `externalSecretsProvider`

Reply:
Updated.